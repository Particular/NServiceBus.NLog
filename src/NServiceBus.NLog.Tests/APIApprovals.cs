using NServiceBus;
using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(NLogFactory).Assembly);
        Approver.Verify(publicApi);
    }
}