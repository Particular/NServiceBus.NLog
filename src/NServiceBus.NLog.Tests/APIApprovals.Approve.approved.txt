[assembly: System.CLSCompliantAttribute(true)]
[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
namespace NServiceBus
{
    public class NLogFactory : NServiceBus.Logging.LoggingFactoryDefinition
    {
        public NLogFactory() { }
        protected override NServiceBus.Logging.ILoggerFactory GetLoggingFactory() { }
    }
}